ecdsa.go

```diff
@@ -62,7 +62,7 @@ func (m *SigningMethodECDSA) Verify(signingString string, sig []byte, key interf
 	case *ecdsa.PublicKey:
 		ecdsaKey = k
 	default:
-		return ErrInvalidKeyType
+		return newError("ecdsa verify expects *ecsda.PublicKey", ErrInvalidKeyType)
```

Comment:
Minor thing. Should we uppercase `ECDSA` and the others so that it matches the signing method? Otherwise, it looks good.